packages/react-component-library/src/components/SelectE/SelectE.test.tsx

```diff
@@ -266,4 +275,52 @@ describe('SelectE', () => {
       expect(wrapper.getByTestId('select-input')).toHaveValue('')
     })
   })
+
+  describe('when options have icons and badges', () => {
```

Comment:
Might be nice to test icons and badges separately if they don't depend on each other.